src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -42,10 +44,26 @@ public String saveTicket(Ticket ticket){
             log.info(e.toString());
             return "error";
         }
-
         return "start";
     }
 
+    @RequestMapping("/uploadImage")
+    public String uploadImage( @RequestParam("imageFile") MultipartFile imageFile, Model model) {
+        String  returnValue = "start";
+        try {
+            ticketService.saveImage(imageFile);
+            Ticket ticket = new Ticket();
+            model.addAttribute( "ticket", ticket);
+            returnValue = "start";
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
While catch blocks are a good idea, use proper logging instead of printStackTrace().
- Logging has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.